docs/sources/search/_index.md

```diff
@@ -83,3 +83,18 @@ To filter dashboard search result by a tag, complete one of the following steps:
   All tags will be shown, and when you select a tag, the dashboard search will be instantly filtered.
 
 > **Note:** When using only a keyboard, press the `tab` key and navigate to the **Filter by tag** drop-down menu, press the down arrow key `▼` to activate the menu and locate a tag, and press `Enter` to select the tag.
+
+## Command Palette
+
+By typing in this field, you can:
+
+1. Search and open dashboards and folders.
+2. Create dashboards and alert rules.
```

Comment:
```suggestion
- Create dashboards and alert rules.
```

Reply:
Hi @chri2547! I'm curious about why I shouldn't use the final dot in this case. Would you mind explaining it to me?